recipes/server.rb

```diff
@@ -114,3 +115,24 @@
   notifies :restart, 'service[graylog-server]', node['graylog2']['restart'].to_sym
   only_if { node.default['graylog2']['major_version'].to_f <= 2.2 }
 end
+
+ruby_block 'set access token' do
```

Comment:
This should be implemented as a Chef provider. We had a very simple API support a long time ago. It never supported big parts of the API so we decided to ditch it. Take a look here: https://github.com/Graylog2/graylog2-cookbook/blob/9b182feda64899bd7876bac2a970dfcad9757c32/providers/token.rb
and here:
https://github.com/Graylog2/graylog2-cookbook/tree/9b182feda64899bd7876bac2a970dfcad9757c32/providers

Reply:
These links are very useful. Thank you!